edb/schema/objects.py

```diff
@@ -1123,6 +1123,21 @@ def compare_field_value(
             assert isinstance(result, (float, int))
             return result
 
+        # If a field is not inheritable (and thus cannot be affected
```

Comment:
Please move this check into `compare_obj_field_value` (the `if explicit` branch), for clarity.

Reply:
I moved the logic to `compare_obj_field_value`.